lib/prompts/expand.js

```diff
@@ -222,11 +222,13 @@ Prompt.prototype.validateChoices = function (choices) {
  * @return {String} The rendered choices key string
  */
 Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
-  var defIndex = 0;
+  var defStr = this.opt.choices.pluck('key');
```

Comment:
@SBoudrias The reason to use `this.opt.choices.pluck('key')` is to get choices with `keys` excluding separators. 
`this.opt.choices.length` === 6 
`defStr.length` === 5